### PR TITLE
Stage plan

### PR DESCRIPTION
**Pull request by abotzki**

### CODE REVIEW

File: pages/lifecycle/plan.md [hunk 1]
```diff
@@ -4,6 +4,66 @@ title: Plan
...
+
+## What are the pedagogical aspects?
+
+TODO: address some comments in the gdoc
+
+- Training needs analysis: Identifying specific training gaps. Assessing pedagogical, technical and logistical requirements.
+- Target audience: Defining the specific group of individuals who will benefit from the training.
+- Learning objectives: Clearly outlining the desired overarching outcomes of the training.
+- Curriculum development: Creating a detailed training curriculum that aligns with learning objectives.
+- Learning paths: Review related learning paths and determine if and how this training aligns with them.
```
Comment (elinkronander) [inline pages/lifecycle/plan.md:33]:
```suggestion
- Learning paths: Review related learning paths and determine if and how this training aligns with them. If the training will be structures as a learning path, ensure to take the available guidelines into account.
```

File: pages/lifecycle/plan.md [hunk 2]
```diff
@@ -4,6 +4,66 @@ title: Plan
...
+TODO: address some comments in the gdoc
+
+- Training needs analysis: Identifying specific training gaps. Assessing pedagogical, technical and logistical requirements.
+- Target audience: Defining the specific group of individuals who will benefit from the training.
+- Learning objectives: Clearly outlining the desired overarching outcomes of the training.
+- Curriculum development: Creating a detailed training curriculum that aligns with learning objectives.
+- Learning paths: Review related learning paths and determine if and how this training aligns with them. 
+- Train the Trainer : Continuously offer training for your trainers. 
+
+TODO: design illustration
```
Comment (elinkronander) [inline pages/lifecycle/plan.md:36]:
```suggestion

```
I've created an issue for this TODO, needs a better description.

File: pages/lifecycle/plan.md [hunk 3]
```diff
@@ -4,6 +4,66 @@ title: Plan
...
+* **Training format and delivery**: Selecting the most appropriate format (e.g., classroom, online, blended) and delivery method and training schedule (e.g.: duration, spread of lessons, time of offering) according to level, audience availability and topic.
+* **Preliminary logistical arrangements**: Planning for venue, equipment, and other available general resources for participants during the training session (e.g. restaurantes, meeting rooms, break rooms, etc …) that might have  logistical requirements in order to establish budget etc. 
+* **Marketing and promotion**: Developing or integrating to already existing  communication strategies to attract and inform potential participants.
+* **Code of Conduct**: Establish a code of conduct for training events to adhere to
+* **Evaluate**: Developing or reusing already existing strategies to l evaluate the training and include that in your plan.
+
+## What are the pedagogical aspects?
+
+TODO: address some comments in the gdoc
```
Comment (elinkronander) [inline pages/lifecycle/plan.md:27]:
```suggestion

```
I've adressed some and added an issue for other's

File: pages/lifecycle/plan.md [hunk 4]
```diff
@@ -4,6 +4,66 @@ title: Plan
...
+- Learning paths: Review related learning paths and determine if and how this training aligns with them. 
+- Train the Trainer : Continuously offer training for your trainers. 
+
+TODO: design illustration 
+![Nicoll's paradidm](assets/img/screenshots/allegra-via-f1000.png)
+
+## Related resources: 
+[ELIXIR-GOBLET Train-the-trainer](train-the-trainer), [Learning Paths](learning-paths), [TeSS](tess), ELIXIR Code of Conduct, FAIR training handbook, How do we identify training needs
+
+TODO: does ELIXIR have anything for identification of training needs (survey?)?
```
Comment (elinkronander) [inline pages/lifecycle/plan.md:42]:
```suggestion

```
Part of issue #158

**Review by elinkronander — COMMENTED** [review]
This looks good apart from the TODO's

File: pages/lifecycle/plan.md [hunk 5]
```diff
@@ -4,6 +4,59 @@ title: Plan
...
+* **Training format and delivery**: Selecting the most appropriate format (e.g., classroom, online, blended) and delivery method and training schedule (e.g.: duration, spread of lessons, time of offering) according to level, audience availability and topic.
+* **Preliminary logistical arrangements**: Planning for venue, equipment, and other available general resources for participants during the training session (e.g. restaurantes, meeting rooms, break rooms, etc …) that might have  logistical requirements in order to establish budget etc. 
+* **Marketing and promotion**: Developing or integrating to already existing  communication strategies to attract and inform potential participants.
+* **Code of Conduct**: Establish a code of conduct for training events to adhere to
+* **Evaluate**: Developing or reusing already existing strategies to l evaluate the training and include that in your plan.
+
+## Related resources: 
+[ELIXIR-GOBLET Train-the-trainer](train-the-trainer), [Learning Paths](learning-paths), [TeSS](tess), ELIXIR Code of Conduct, FAIR training handbook, How do we identify training needs
```
Comment (elinkronander) [inline pages/lifecycle/plan.md:40]:
```suggestion
[ELIXIR-GOBLET Train-the-trainer](train-the-trainer), [Learning Paths](learning-paths), [TeSS](tess), [FAIR training handbook](fair-training-handbook)
```

File: pages/lifecycle/plan.md [hunk 6]
```diff
@@ -4,6 +4,59 @@ title: Plan
...
+
+TODO: does ELIXIR have anything for identification of training needs (survey?)?
+
+## References	
+[Train-the-Trainer Concept on Research Data Management](https://doi.org/10.5281/zenodo.13927613)
```
Comment (elinkronander) [inline pages/lifecycle/plan.md:45]:
```suggestion
- [Train-the-Trainer Concept on Research Data Management](https://doi.org/10.5281/zenodo.13927613)
```
For consistency with other pages

**Comment by abotzki** [comment]
Thank you all for your contributions!

**Pull request merged**